src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/ResolverHandler.java

```diff
@@ -33,7 +33,7 @@
      *
      * @throws IllegalArgumentException is thrown when an IllegalArgumentException is thrown by the registerResolver method for any of the given Resolvers.
      */
-    public static Collection<IToken<?>> registerResolvers(final IStandardRequestManager manager, final IRequestResolver... resolvers)
+    public static Collection<IToken<?>> registerResolvers(final IStandardRequestManager manager, final IRequestResolver<?>... resolvers)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1452)